scripts/production/batch_lastz.pl

```diff
@@ -83,15 +83,17 @@
 
 # calculate number of jobs per-mlss
 print STDERR "Estimating number of jobs for each method_link_species_set..\n";
-my (%mlss_job_count, %chunk_counts, %chain_dnafrag_counts, %chain_job_count, %dnafrag_interval_counts);
-# my %total_job_count;
+my (%mlss_job_count, %chunk_counts, %chain_job_count, %dnafrag_interval_counts);
 foreach my $mlss ( @current_lastz_mlsses ) {
     my ($mlss_gdbs, $ref_chunk_count, $non_ref_chunk_count, $filter_dups_job_count);
     if ( defined $mlss->get_tagvalue('species_set_size') ) {
         $mlss_gdbs = $mlss->species_set->genome_dbs;
         $ref_chunk_count = get_ref_chunk_count($mlss_gdbs->[0]);
         $non_ref_chunk_count = get_non_ref_chunk_count($mlss_gdbs->[0]);
+        # For polyploid self-alignments the computations should be times 3
+        # instead of 2, but they are separated in their own batch anyway
```

Comment:
In the end, do self-alignments really have to be in their own batch ? I was quite weary of mixing things, esp. as the code had not been tested. But now I think we could mix them with the rest.

Reply:
Should update the formula to be based on the number of components

Reply:
Ok, I will work on a new formula to be more accurate in the polyploid cases.